BeerApp/src/main/java/beerapp/dal/BeersCountDao.java

```diff
@@ -0,0 +1,54 @@
+package beerapp.dal;
+
+import static beerapp.dal.Utility.safeCloseResultSet;
+
+import beerapp.model.Beer;
+import beerapp.model.BeersCount;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public enum BeersCountDao {
+  INSTANCE;
+  private ConnectionManager connectionManager;
+
+  private BeersCountDao() {
+    connectionManager = new ConnectionManager();
+  }
+
+  /**
+     * Get a list of beerNames whose style contains "Lager" with the top ten number of reviews
+     */
+    public List<BeersCount> getTopBeersCountsByLager(int topNum) {
+      List<BeersCount> resultsReview = new ArrayList<>();
```

Comment:
we actually just need a `List<String>` that has a list of beer names, so the new class `BeerCount` is not necessary. Could you refactor the code?

Reply:
I deleted the class `beerCount` and make some changes on `BeerCountDao` and `GetTopBeersCountsByLager`